core/arch/arm/kernel/arch_scall.c

```diff
@@ -94,9 +95,14 @@ static void save_panic_regs_a32_ta(struct thread_specific_data *tsd,
 static void save_panic_regs_a64_ta(struct thread_specific_data *tsd,
 				   uint64_t *pushed)
 {
+	uint64_t x29, elr;
```

Comment:
Please declare one local variable per line and initialize them.

Reply:
updated